src/Mailgun/Api/Domains.php

```diff
@@ -0,0 +1,270 @@
+<?php
+
+/*
+ * Copyright (C) 2013-2016 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+namespace Mailgun\Api;
+
+use Mailgun\Assert;
+use Mailgun\Resource\Api\Domains\ComplexDomain;
+use Mailgun\Resource\Api\Domains\CredentialListResponse;
+use Mailgun\Resource\Api\Domains\DeliverySettingsResponse;
+use Mailgun\Resource\Api\Domains\DomainResponse;
+use Mailgun\Resource\Api\Domains\DomainListResponse;
+use Mailgun\Resource\Api\Domains\SettingsUpdateResponse;
+use Mailgun\Resource\Api\SimpleResponse;
+
+/**
+ * {@link https://documentation.mailgun.com/api-domains.html}
+ *
+ * @author Sean Johnson <sean@mailgun.com>
+ */
+class Domains extends AbstractApi
+{
+    /**
+     * Returns a list of domains on the account.
+     *
+     * @param int $limit
+     * @param int $skip
+     *
+     * @return DomainListResponse
+     */
+    public function list($limit = 100, $skip = 0)
+    {
+        Assert::integer($limit);
+        Assert::integer($skip);
+
+        $params = array(
+            'limit' => $limit,
+            'skip'  => $skip,
+        );
+
+        $response = $this->get('/v3/domains', $params);
+
+        return $this->serializer->deserialize($response, DomainListResponse::class);
+    }
+
+    /**
+     * Returns a single domain.
+     *
+     * @param string    $domain     Name of the domain.
```

Comment:
There are no other `@param` to this should be:
`@param string $domain Name of the domain.`